src/Services/SwaggerService.php

```diff
@@ -605,7 +626,40 @@ public function saveProductionData()
 
     public function getDocFileContent()
     {
-        return $this->driver->getDocumentation();
+        $this->documentation = $this->driver->getDocumentation();
+
+        $additionalDocs = config('auto-doc.additional_paths', []);
+
+        foreach ($additionalDocs as $filePath) {
+            $fileContent = json_decode(file_get_contents($filePath), true);
+
+            $paths = array_keys($fileContent['paths']);
+
+            foreach ($paths as $path) {
+                if (empty($this->documentation['paths'][$path])) {
+                    $this->documentation['paths'][$path] = $fileContent['paths'][$path];
```

Comment:
let's move `$fileContent['paths'][$path]` to the separate variable

```suggestion
                    $this->documentation['paths'][$path] = $additionalDocPatch;
```